packages/d3fc-webgl/src/buffers/float32ArrayFactory.js

```diff
@@ -0,0 +1,12 @@
+function float32ArrayFactory(requiredLength) {
```

Comment:
I'm not sure this is doing what you expect it to. The `cachedArray` is effectively global.

Reply:
Thanks, makes more sense now, changed.